From d30c3d54ae0dec1197970c090b0b688eec38838c Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 22 Nov 2011 15:53:43 +0000
Subject: [PATCH 064/462] replacement for trace.dll, trchdl.dll and
 trcserv.exe: revert (trace.dll with configuration does not work due to corba
 problems)

git-svn-id: svn://localhost/SelfServiceBuild/trunk@88 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.mak | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/cpp/makefile.mak b/SelfServiceBuild/Build/cpp/makefile.mak
index 8ccdd0e..7f165a6 100644
--- a/SelfServiceBuild/Build/cpp/makefile.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.mak
@@ -41,7 +41,8 @@ COMMON_DIRECTORIES= \
  Massai\cpp\CfgClnt\static \
  Massai\cpp\CfgXMgrB \
  Massai\cpp\CfgXMgrB\static \
- Massai\cpp\Trace \
+ Services\cpp\TrcServ \
+ Services\cpp\TrcServ\static \
  Massai\cpp\SecureLogging \
  Massai\cpp\CfgTools \
  Massai\cpp\CfgTools\static \
-- 
2.41.0.windows.1

